dl.html

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
+<meta http-equiv="refresh" content="1; url=en/download">
```

Comment:
```suggestion
<meta http-equiv="refresh" content="1; url=https://delta.chat/en/download">
```

Can we use an absolute URL here? it kinda breaks the preview, but e.g. lynx (non-js command line browser) only uses these refresh URLs for absolute URLs.